dlp/dlp_inspect_image_file2.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def inspect_image_file project_id, filepath
```

Comment:
FYI – this is the canonical example I'm using

https://github.com/beccasaurus/gapic-docs-samples/blob/master/code-samples/canonical-samples/ruby/synchronous_api_request.rb

Reply:
@jmdobry